@navikt/core/react/src/accordion/Accordion.tsx

```diff
@@ -31,10 +31,12 @@ interface AccordionComponent
 
 export interface AccordionProps extends React.HTMLAttributes<HTMLDivElement> {
   /**
+   * @deprecated Prop will be removed in future versions. "default" will be the only variant.
    * @default "default"
    */
   variant?: "default" | "neutral";
   /**
+   * @deprecated Prop will be removed and replaced by `size` in future versions.
    * @default "small"
    */
   headingSize?: "large" | "medium" | "small" | "xsmall";
```

Comment:
Should we delay deprecating `headingSize` until new design is launched? People will not be able to remove the prop until then anyways (without having to live with undesirable heading size temporarily). Maybe we can do something clever and map `headingSize` to `size` temporarily? 🤔